LinkResolverService.cs

```diff
@@ -86,13 +87,17 @@ public class LinkResolverService
                     target = targetFrame.FrameName;
                 }
 
-
-                href = ResolvePagedataHref(pageData, options);
+                href = ResolvePagedataHref(pageData, options, languageBranchId);
             }
             else
             {
                 href = UrlResolver.Service.GetUrl(content);
 
+                if (!string.IsNullOrWhiteSpace(languageBranchId))
```

Comment:
Can we change this so we're not potentially resolving the URL twice? Something like:
`href = string.IsNullOrWhiteSpace(languageBranchId) ? UrlResolver.Service.GetUrl(content) : UrlResolver.Service.GetUrl(content.ContentLink, languageBranchId)`?